### ISSUE TITLE
Update URL in “Which sites use Django” FAQ

### ISSUE THREAD

**Issue by richcodesnothing** [django/djangoproject.com#1490]
DjangoSites.org has shut down as of January 2024. It is listed as a good resource that contains examples of Django web apps in the “Which sites use Django” FAQ. Ross Poulton managed that site for years and recently published a post on the site explaining why he has taken it down. He is pointing visitors to https://builtwithdjango.com/projects/, which is built and actively managed by Rasul Kireev. We should update the FAQ with Rasul’s site and any other the community believes best showcases Django. ❤️

**Comment by spulloor** [comment]
As this is a small change can i take up this issue.

**Comment by adamzap** [comment]
I think this was already done in 88451586: https://docs.djangoproject.com/en/5.1/faq/general/#which-sites-use-django

Can we close this issue?

**Comment by bmispelon** [comment]
Good call, the link was changed on the site in #1471 and was removed from the Django FAQ a few months later.
